### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by normainobary2021**
In this PR I did the following:
- Deleted all the boilerplate from CRA (text, images, styles).
- Created a directory called components.
- Created a new Calculator.js file inside components folder.
- In Calculator.js, I created a React component that matches this design.
- Used class-based components.
- Removed all linter errors.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+              >
+                /
+              </button>
+              <button
+                type="button"
+                className="key multiply"
+              >
+                *
+              </button>
+              <button
```
Comment (RolandM99) [inline src/components/Calculator.js]: - [ ] The `Division` and `Multiplication` operations should have the same symbols as on the original design provided. division: `÷` and multiplication: `×`

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi  @normainobary2021 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (normainobary2021) [reply src/components/Calculator.js]: Thank you for that review @RolandM99

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @normainobary2021👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by normainobary2021** [comment]
Thank you @thecodechaser for that kind review

**Pull request merged**